src/Livewire/Product/ProductPropertyGroupList.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace FluxErp\Livewire\Product;
+
+use FluxErp\Livewire\DataTables\ProductPropertyGroupList as BaseProductPropertyGroupList;
+
+class ProductPropertyGroupList extends BaseProductPropertyGroupList
+{
+    public bool $showFilterInputs = false;
+
+    public ?bool $isSearchable = true;
+
+    public bool $hasSidebar = false;
+
+    public function startSearch(): void
```

Comment:
Why not searchable on the model with collection driver?

Reply:
This was just copied from the ProductOptionGroupList which is used in the create variants.